document/chapters/03-sdf_preconditioner.tex

```diff
@@ -1183,6 +1183,106 @@ \subsection{Memory Usage}
 when creating DAGMC problem input, particularly if the simulation is going to be
 run in parallel.
 
+\section{Other Distance Limit Optimizations}
+
+DAGMC supports a mode in which the distance to the next physics event is used to
+limit the length of a ray and optimize the BVH traversal process. Nodes which
+the ray does not reach are not visited, reducing the average traversal time of
+rays in the BVH. If the ray does not intersect a surface, DAGMC returns a next
+surface distance greater than that of the distance to the next physics event. 
+The physics kernel is agnostic to this change, and comparisons of these
+distances are used as they are normally to determine a particle's next event.
+
+This distance limit optimization is only guaranteed to be robust for fully
+sealed DAGMC geometries. Because rays are not required to return a surface
+intersection in this mode, ray queries moving into gaps of the unsealed model
+will update the particle's position to the next physics event. If the next
+physics event is outside of the volume, particles will exit the volume without
+error in the particle tracking algorithm. In this scenario, particles can
+numerically leave the volume without updating the material in which the particle
+is interacting in the physics kernel, resulting in inaccurate physics
+events. This will continue until the particle is killed due to its low
+statistical contribution to the simulation solution. Occasional
```

Comment:
Note: this was first discovered in a problem with a multiplying medium, and the particle filled the bank because it leaked out of a fissile region and looked like super-critical system.

Reply:
Should I add this as an example of the consequences for this pathology?

Reply:
Not sure you need to itemize them all, but I wanted you to know that there are multiple failure modes.  You can choose how you want to summarize it all...